packages/happy-dom/src/nodes/html-input-element/HTMLInputElementDateUtility.ts

```diff
@@ -14,8 +14,8 @@ export default class HTMLInputElementDateUtility {
 		date.setUTCDate(date.getUTCDate() - day + 3);
 		const firstThursday = date.getTime();
 		date.setUTCMonth(0, 1);
-		if (date.getDay() !== 4) {
```

Comment:
It seems like we are using this:
https://stackoverflow.com/a/16591175/12415847

But we should use this (it seems like JSDOM is using this one as well):
https://stackoverflow.com/a/6117889/12415847